module/applications/components/filigree-box.mjs

```diff
@@ -29,85 +41,67 @@ export default class FiligreeBoxElement extends HTMLElement {
 
   /* -------------------------------------------- */
 
-  /**
-   * The stylesheet to attach to the element's shadow root.
-   * @type {CSSStyleSheet}
-   * @protected
-   */
-  static _stylesheet;
+  /** @inheritDoc */
+  _adoptStyleSheet(sheet) {
+    this.#shadowRoot.adoptedStyleSheets = [sheet];
+  }
 
   /* -------------------------------------------- */
 
-  /**
-   * Build the shadow DOM's styles.
-   */
-  #buildStyle() {
-    if ( !this.constructor._stylesheet ) {
-      this.constructor._stylesheet = new CSSStyleSheet();
-      this.constructor._stylesheet.replaceSync(`
-        :host {
-          position: relative;
-          isolation: isolate;
-          min-height: 56px;
-          filter: var(--filigree-drop-shadow, drop-shadow(0 0 12px var(--dnd5e-shadow-15)));
-        }
-        .backdrop {
-          --chamfer: 12px;
-          position: absolute;
-          inset: 0;
-          background: var(--filigree-background-color, var(--dnd5e-color-card));
-          z-index: -2;
-          clip-path: polygon(
-            var(--chamfer) 0,
-            calc(100% - var(--chamfer)) 0,
-            100% var(--chamfer),
-            100% calc(100% - var(--chamfer)),
-            calc(100% - var(--chamfer)) 100%,
-            var(--chamfer) 100%,
-            0 calc(100% - var(--chamfer)),
-            0 var(--chamfer)
-          );
-        }
-        .filigree {
-          position: absolute;
-          fill: var(--filigree-border-color, var(--dnd5e-color-gold));
-          z-index: -1;
+  /** @inheritDoc */
+  _buildCSS(sheet) {
```

Comment:
I think this could be a simple `static CSS = `, but I was trying to avoid using statics at all due to this note:

> We also can no longer use a static variable to store our CSS obj, since in Chrome based browsers the static element is shared across windows (though not the case in firefox). So we need to keep a map from document to CSS obj singleton and do the lookup manually.

Even avoiding it though, it seems it was not working in Firefox. If we can get to the bottom of what's happening there, we can revisit this design hopefully.